private/bufpkg/bufprotoplugin/cmd/protoc-gen-go-apiclient/main.go

```diff
@@ -95,6 +95,27 @@ func handleGlobal(helper protogenutil.NamedHelper, plugin *protogen.Plugin, goPa
 	if err != nil {
 		return err
 	}
+	g.P(`const (`)
```

Comment:
I put the `ProviderScheme` code here in the `apiclient` plugin since I figured this was more global.  I can move it though if this isn't the best place.